docs/source/index.rst

```diff
@@ -99,8 +99,9 @@ Welcome to Kedro's documentation!
    06_nodes_and_pipelines/01_nodes
    06_nodes_and_pipelines/02_pipeline_introduction
    06_nodes_and_pipelines/03_modular_pipelines
-   06_nodes_and_pipelines/04_run_a_pipeline
-   06_nodes_and_pipelines/05_slice_a_pipeline
+   06_nodes_and_pipelines/04_micro_packaging
+   06_nodes_and_pipelines/05_run_a_pipeline
+   06_nodes_and_pipelines/06_slice_a_pipeline
```

Comment:
Personally I would say micropackaging should be last, as running and slicing is more important / frequently used on a daily basis. 

Also bare in mind this will conflict on develop and we'll have to remove the numbering, but the _order_ in `index.rst` matters.

Reply:
When the other stuff is settled I will sync up with you re this point and the develop question